config/blobs.yml

```diff
@@ -2,6 +2,9 @@ jmeter/apache-jmeter-3.0_1.tgz:
   size: 45178166
   object_id: e28d6f50-1c64-41ca-6352-2dcffdbb9894
   sha: 0063516826add27a49eb391e61a6a60dc8065813
+jmeter/apache-jmeter-5.0_rmi.tgz:
+  size: 53908732
+  sha: 9df3ec29e6c7f64b396fbffcf1ae4c70128d6f01
```

Comment:
Thanks for the change to upgrade jmeter to version 5.
I think it will be easier for me to do the change myself as it is semi-automated from my side.

Would you mind removing the changes for upgrading jmeter from this pull request so that its only focus is the `user.properties` change ?

Reply:
Yes, please.